utils/azure_users.py

```diff
@@ -185,3 +186,32 @@ def get_role_data(self, role_id, is_grant=True):
             return {field_name: field_value}
         else:
             return {field_name: None}
+
+    def _get_user(self, user_id):
+        endpoint = "{endpoint}/users/{user_id}?api-version=1.6".format(
+            endpoint=self.config.ENDPOINT, user_id=user_id
+        )
+        response = requests.get(endpoint, auth=BearerAuth(self.access_token))
+        assert 200 == response.status_code
+        return response.json()
+
+    def is_test_user(self, user_id):
+        response = self._get_user(user_id)
+        field_name, field_value = ROLE_FIELD_VALUES['test']
+        return field_name in response and field_value == response[field_name]
+
+    def _get_test_user_ids(self):
+        field_name, field_value = ROLE_FIELD_VALUES['test']
+        endpoint = "{endpoint}/users?api-version=1.6&$select=objectId,{field_name}&$filter={field_name} eq '{field_value}'".format(
+            endpoint=self.config.ENDPOINT,
+            field_name=field_name,
+            field_value=field_value,
+        )
+        response = requests.get(endpoint, auth=BearerAuth(self.access_token))
+        assert 200 == response.status_code
+        assert 'value' in response.json()
+        return response.json()['value']
+
+    def get_test_user_ids(self):
```

Comment:
I just left the same comment in Robert's PR. Why do we have this "name duplicated" method

Reply:
With Roberto, we have rethought it and we have united it in a single method so as not to have these repeated functions.